nixos/modules/image/repart.nix

```diff
@@ -95,6 +95,40 @@ in
       '';
     };
 
+    compression = lib.mkOption {
+      description = "Image compression";
+      default = { };
+      type = lib.types.submodule ({ config, ... }: {
+        options = {
+          enable = lib.mkEnableOption (lib.mdDoc "Image compression");
```

Comment:
I think it would be good to have compression enabled by default, similar to how the sd-image module does.

Reply:
I'm against this because whenever you need the files for testing, you need them uncompressed. Uncompressing them first in every test leads to a lot of boilerplate and ultimately to more large images ending up in the cache.

Privately, I've compressed the images by default too but have reverted back to uncompressed now because of this reason.

Reply:
Fair enough!